lib/injector.rb

```diff
@@ -0,0 +1,11 @@
+class Array
+
+  def injector &lambda
+  	len = self.length - 1
+    result = self[0]
+    (1..len).each do |x|
+	  result = yield result, self[x]
+	end
```

Comment:
`end` at 8, 1 is not aligned with `(1..len).each do |x|` at 6, 4<br>Tab detected.